src/index.ts

```diff
@@ -79,6 +79,9 @@ export default function makeWebSocketObservable<T extends WebSocketPayload = Web
     socket.onopen = () => {
       isSocketOpen = true
       observer.next(getWebSocketResponses)
```

Comment:
Okay I get this now, sorry for the confusion. Could you put this `next` in an else branch of the next condition, there's no point issuing that event when the stream is closed.

Reply:
Actually, it won't fire any event since the observer has unsubscribed, but it makes the code clearer